src/runner/main.cpp

```diff
@@ -138,6 +138,7 @@ int runner(bool isProcessElevated)
         std::unordered_set<std::wstring> module_folders = {
             L"",
             L"FileExplorerPreview/",
+            L"FancyZones"

```

Comment:
Is it ok that in previous line slash added after module name, and in this line it isn't?

Reply:
Even if it works we should be consistent, one of those two lines has to change.

Reply:
I had build errors in PowerToysSetup with not found files `...modules\FancyZones\*.dll`. 
Added slash fixed that